torch/testing/_internal/common_methods_invocations.py

```diff
@@ -108,7 +123,9 @@ def __init__(self,
                  promotes_integers_to_float=False,  # whether op promotes unary output to float or not
                  sample_inputs_func=None,  # function to generate sample inputs
```

Comment:
I would expect "sparse_sparse_inputs_func" to appear here (note slight name change for consistency)